### PR TITLE
chore(docs): add docker-compose instructions

### PR DESCRIPTION
**Pull request by alrevuelta**
- [x] Add docker-compose instructions + update main readme.

### CODE REVIEW

File: docs/operators/quickstart.md [hunk 1]
```diff
@@ -36,6 +36,20 @@ docker run -i -t -p 60000:60000 -p 9000:9000/udp \
...
 
+## Option 3: run nwaku with docker compose
+
+*Prerequisites: `docker` and `docker-compose`*.
+Allows to run `nwaku` with `prometheus` and `grafana`, with an already provisioned dashboard in few simple steps. See [nwaku-compose](https://github.com/alrevuelta/nwaku-compose).
+
+```bash
+git clone https://github.com/alrevuelta/nwaku-compose.git
+cd nwaku-compose
+export MY_EXT_IP=$(dig TXT +short o-o.myaddr.l.google.com @ns1.google.com | awk -F'"' '{ print $2}')
```
Comment (rymnc) [inline docs/operators/quickstart.md:47]: Maybe add additional ways to fetch/set the external IP?

**Review by rymnc — APPROVED** [review]
LGTM

File: docs/operators/quickstart.md [hunk 2]
```diff
@@ -36,6 +36,20 @@ docker run -i -t -p 60000:60000 -p 9000:9000/udp \
     --nat:extip:[yourpublicip] # or, if you are behind a nat: --nat=any
 ```
 
+## Option 3: run nwaku with docker compose
+
+*Prerequisites: `docker` and `docker-compose`*.
+Allows to run `nwaku` with `prometheus` and `grafana`, with an already provisioned dashboard in few simple steps. See [nwaku-compose](https://github.com/alrevuelta/nwaku-compose).
```
Comment (jm-clius) [inline docs/operators/quickstart.md:42]:
```suggestion
Allows to run `nwaku` with `prometheus` and `grafana`, with an already provisioned dashboard, in a few simple steps.
See [nwaku-compose](https://github.com/alrevuelta/nwaku-compose).
```

Added sembr.

File: docs/operators/quickstart.md [hunk 3]
```diff
@@ -36,6 +36,20 @@ docker run -i -t -p 60000:60000 -p 9000:9000/udp \
...
+
+```bash
+git clone https://github.com/alrevuelta/nwaku-compose.git
+cd nwaku-compose
+export MY_EXT_IP=$(dig TXT +short o-o.myaddr.l.google.com @ns1.google.com | awk -F'"' '{ print $2}')
+docker-compose up -d
+```
+
+Go to [http://localhost:3000/d/yns_4vFVk/nwaku-monitoring?orgId=1](http://localhost:3000/d/yns_4vFVk/nwaku-monitoring?orgId=1) and after some seconds, your node metrics will be live there. As simple as that.
```
Comment (jm-clius) [inline docs/operators/quickstart.md:51]:
```suggestion
Go to [http://localhost:3000/d/yns_4vFVk/nwaku-monitoring?orgId=1](http://localhost:3000/d/yns_4vFVk/nwaku-monitoring?orgId=1) and after some seconds, your node metrics will be live there.
As simple as that.
```
Semantic break.

**Review by jm-clius — APPROVED** [review]
Great! Thanks for that.

**Pull request merged**